ehsm_kms_service/secret_manager_apis.js

```diff
@@ -265,9 +265,69 @@ const createSecret = async (res, appid, payload, DB) => {
     }
 }
 
+//update the description of secret
+const updateSecretDesc = async (res, appid, payload, DB) => {
+    //get and check param in payload
+    let secretName = getParam_String(payload, 'secretName')
+    let description = getParam_String(payload, 'description')
+    if (!checkStringParam(secretName, true, SECRETNAME_LENGTH_MAX)) {
+        res.send(_result(400, `secretName cannot be empty, must be string and length not more than ${SECRETNAME_LENGTH_MAX}`))
+        return
+    }
+    if (!checkStringParam(description, false, DESCRIPTION_LENGTH_MAX)) {
```

Comment:
if the description is **undefined**, what happens?

Reply:
Same result as ' '

Reply:
Undefined is equivalent to entering an empty description